cmd/nfd-worker/main.go

```diff
@@ -73,6 +74,7 @@ func main() {
 	utils.ConfigureGrpcKlog()
 
 	// Get new NfdWorker instance
+	args.GrpcHealthPort = GrpcHealthPort
```

Comment:
Just checkin', is this similar to how it's currently handled in nfd-master (i.e. not user configurable)? If so, it's good, we can add command line flag to both nfd-master and nfd-worker later, if needed.